pom.xml

```diff
@@ -14,6 +14,23 @@
 
     <name>icd10-dictionary</name>
     <description>REST Service for ICD10 lookups</description>
+    <url>https://github.com/samply/maven-parent/icd10-dictionary</url>
+
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
```

Comment:
You have forgotten the <scm> and <organisation> tag.
You can find a full example in the pom.xml of the share-client. https://github.com/samply/share-client/blob/master/pom.xml

Reply:
Why do I have to explicitly state these two tags? They are inherited by the Parent-POM (part of effective POM):
`
  <organization>
    <name>Samply Community</name>
  </organization>

...

  <scm>
    <connection>scm:git:https://github.com/samply/maven-parent.git/icd10-dictionary</connection>
    <developerConnection>scm:git:https://github.com/samply/maven-parent.git/icd10-dictionary</developerConnection>
    <url>https://github.com/samply/maven-parent/icd10-dictionary</url>
  </scm>
`